src/utils/Processing.ts

```diff
@@ -308,6 +308,8 @@ const IGNORED_NON_RESOURCE_DIRECTORIES = [
 ];
 
 export type ProcessingOptions = {
+  indent?: boolean;
+  metaProfile?: 'only-one' | 'first' | 'none';
   [key: string]: boolean | number | string;
```

Comment:
Why do the `ProcessingOptions` need the
```javascript
  [key: string]: boolean | number | string;
```
line? I'm not seeing where that is used.

Reply:
This is mostly just some anticipatory implementation on my part, where we might want a way to set additional primitive values so that they could be used in custom optimizers. Right now, the only place they're actually used is in a few of the `Processing` tests.